source/ShipInfoPanel.cpp

```diff
@@ -169,9 +169,28 @@ bool ShipInfoPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 	else if(panelState.CanEdit() && key == 'D')
 	{
 		if(shipIt->get() != player.Flagship())
-			GetUI()->Push(new Dialog(this, &ShipInfoPanel::Disown, "Are you sure you want to disown \""
+		{
+			map<const Outfit*, int> uniqueOutfits;
+			for(const auto &it : shipIt->get()->Outfits())
+				if(it.first->Category() == "Unique" || it.first->Attributes().Get("unique") > 0)
+					uniqueOutfits[it.first] = it.second;
```

Comment:
```suggestion
					uniqueOutfits[it.first] += it.second;
```

Reply:
You can disown only one ship at a time, so it's not needed.